content/rma_intro.tex

```diff
@@ -16,31 +16,40 @@
 one-sided communication interfaces via \Celev{} generic selection 
 (\Celev{} \S6.5.1.1\footnote{Formally, the \Celev{} specification is ISO/IEC 9899:2011(E).})
 for block, scalar, and block-strided put and get communication. 
-Such type-generic routines are supported for the ``standard \ac{RMA} types’’
-identified in Table \ref{stdrmatypes}.
+Such type-generic routines are supported for the ``standard \ac{RMA} types''
+listed in Table \ref{stdrmatypes}.
 
 \begin{table}[h]
   \begin{center}
     \begin{tabular}{|l|l|}
       \hline
-      \TYPE & \TYPENAME\\
-      \hline
-      float & float\\
-      \hline
-      double & double\\
-      \hline
-      long double & longdouble\\
-      \hline
-      char & char\\
-      \hline
-      short & short\\
-      \hline
-      int & int\\
-      \hline
-      long & long\\
-      \hline
-      long long & longlong\\
-      \hline
+      \TYPE              & \TYPENAME  \\ \hline
+      float              & float      \\ \hline
+      double             & double     \\ \hline
+      long double        & longdouble \\ \hline
+      char               & char       \\ \hline
+      signed char        & schar      \\ \hline
+      short              & short      \\ \hline
+      int                & int        \\ \hline
+      long               & long       \\ \hline
+      long long          & longlong   \\ \hline
+      unsigned char      & uchar      \\ \hline
+      unsigned short     & ushort     \\ \hline
+      unsigned int       & uint       \\ \hline
+      unsigned long      & ulong      \\ \hline
+      unsigned long long & ulonglong  \\ \hline
+      int8\_t            & int8       \\ \hline
+      int16\_t           & int16      \\ \hline
+      int32\_t           & int32      \\ \hline
+      int64\_t           & int64      \\ \hline
+      uint8\_t           & uint8      \\ \hline
+      uint16\_t          & uint16     \\ \hline
+      uint32\_t          & uint32     \\ \hline
+      uint64\_t          & uint64     \\ \hline
+      intptr\_t          & intptr     \\ \hline
+      uintptr\_t         & uintptr    \\ \hline
```

Comment:
The ```inptr_t``` and ```uintptr_t``` types kind of beg the question -- why would one be sending pointers across the network in the first place (and for that matter pointers converted to ints)?  Especially since we caution users against sharing pointers elsewhere in the spec.  I don't have much of an argument against, I just can't imagine any legitimate usage for these types through the SHMEM API.  Maybe someone can help me out?